apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -185,11 +185,13 @@ export const statistics = {
 		);
 
 		// Number of custom fields
-		statsPms.push(
-			LivechatCustomField.col.count().then((count) => {
-				statistics.totalCustomFields = count;
-			}),
-		);
+		statsPms.push((statistics.totalCustomFields = await LivechatCustomField.countDocuments({})));
+
+		// Number of public custom fields
+		statsPms.push((statistics.totalLivechatPublicCustomFields = await LivechatCustomField.countDocuments({ public: true })));
+
```

Comment:
I think is out of scope, but I'd at least write a TODO comment asking for writing a new method on `LivechatCustomField` to get this information in a single query instead of two `$count` aggregations.

Reply:
I agree, it is probably quite simple but I don't think we need to overcomplicate this task